requirements.txt

```diff
@@ -1,4 +1,6 @@
-# This can not be installed from pip, on Fedora you need to install rpm-python{,3}
+# This can not be installed from pip, on Fedora you need to install
+# python{2,3}-rpm RPM including rpm-python pip package.
```

Comment:
@nforro  Thanks for the review.
Updated as your way. 
If it looks good, please merge with squash to 1 commit.